src/app/Models/Traits/SpatieTranslatable/HasTranslations.php

```diff
@@ -213,4 +204,51 @@ public function __call($method, $parameters)
                 break;
         }
     }
+
+    /**
+     * @param  \Illuminate\Database\Eloquent\Model  $model
+     * @param  array  $attributes
+     * @param  string  $locale
+     * @return \Illuminate\Database\Eloquent\Model
+     */
+    private static function fillModelWithTranslations($model, array $attributes, string $locale)
```

Comment:
Hmm... I don think it's appropriate to make this a static method... If it requires the `$model` as a parameter, it kind of defeats the purpose of static methods... those would work without an instance.

Is there a reason for it to be static, that I'm missing here Pedro?

Otherwise, I say let's make this 
```suggestion
    private function fillModelWithTranslations(array $attributes, string $locale)
```

and use `$this` inside it. Which would also clean up the code in `create()` and `update()` a bit, too. No that it needed it... very clean code in this PR (👏👏👏 )... but yeah.

Reply:
We're talking about our own custom methods here, aren't we? Not ::create.

Reply:
Sorry, as you can see, I was specifically talking about create. Kindly disregard, if you didn't know about this static interchange in Laravel model with Builder now you know 👍